Sources/UI/Extensions/UIKit/ViewController+ImagePicker.swift

```diff
@@ -53,7 +59,7 @@ extension ViewController {
         
         let delegate = ImagePickerDelegate(completion) {
             objc_setAssociatedObject(self, delegateKey, nil, .OBJC_ASSOCIATION_RETAIN_NONATOMIC)
-            completion(nil, .notDetermined)
+            // Completion is not called when user cancels. This is intended
```

Comment:
If we want to call completion, it adds complexity to the flow: what do we call it with? `.success(nil)` or `.failure(.noImageSelected)`? `.noImageSelected` can't be an `ImagePickerError` since it's not actually an error, and making Success value an Optional is not beautiful.

Reply:
`failure(.userCancelled)` ?

Reply:
I also thought about that, the question is "is `userCancelled` a failure/error?"
We don't need the completion block in our UI SDK when user cancels (no side effect of this), I can see how some people would need this, but I'd say let's KISS and add it if asked for. It's a better UX when cancelling an action doesn't create a side-effect too.

Reply:
It's not an error but I think it's a failure of a certain action. However, I agree with you "let's wait if someone actually needs it". 👍